src/main/java/org/scijava/script/ScriptInfo.java

```diff
@@ -478,4 +489,27 @@ else if ("value".equalsIgnoreCase(key)) {
 		}
 	}
 
+	/**
+	 * Read entire contents of a Reader and return as String.
+	 *
+	 * @param reader {@link Reader} whose contents should be returned as String.
+	 *          Expected to never be <code>null</code>.
+	 * @return contents of reader as String.
+	 * @throws IOException If an I/O error occurs
+	 * @throws NullPointerException If reader is <code>null</code>
+	 */
+	private static String getReaderContentsAsString(final Reader reader)
+		throws IOException, NullPointerException
+	{
+		final char[] buffer = new char[8192];
```

Comment:
This number is pretty arbitrary. Any ideas on that? Use the probably also arbitrary (but alot bigger) `PARAM_CHAR_MAX` instead? :)

This code is based off of the code I wrote in scripting-java a while ago.